libs/template/cli/src/commands/create.spec.ts

```diff
@@ -0,0 +1,467 @@
+import * as prompts from '@clack/prompts';
+import fs, { PathLike } from 'fs';
+import path from 'path';
+import { expect, it, vi } from 'vitest';
+import { CreateCliCommand, OryxPreset } from './create';
+
+vi.mock('@clack/prompts');
+vi.mock('fs');
+vi.mock('path');
+
+describe('CreateCliCommand', () => {
+  class MockCliArgsService {
+    get = vi.fn();
+  }
+
+  class MockNodeUtilService {
+    downloadFile = vi.fn();
+    extractZip = vi.fn();
+    copyFolder = vi.fn();
+  }
+
+  function setup() {
```

Comment:
So we agreed using setup everywhere?

Reply:
It's been using in the i18n and auth so I think we are ok with using this style and beforeEach?